app/controllers/application_controller.rb

```diff
@@ -177,8 +177,11 @@ def current_cohort(current_user = @current_user, context = @context)
       end
       if context
         current_user.enrollments.active.where(:course_id => context.id).each do |enrollment|
-          enrollment.course_section.students.active.each do |student|
-            res[student.id] = student.name
+          # FIXME: remove Test Student too
+          if enrollment.course_section.name.include?("(")
```

Comment:
Yikes. You should comment this at the very least. Super hack.

Reply:
Can you also queue up a task to write a unit test around this piece of code? When someone inevitable complains that something is not working, it would be nice to see that the code was written to only handle cohorts named with a ```(``` in them (and the justification for why)